UnityProject/Assets/Scripts/Core/Input System/InteractionV2/Validations.cs

```diff
@@ -741,4 +741,18 @@ public static bool HasTarget(TargetedInteraction interaction)
 	{
 		return interaction.TargetObject != null;
 	}
+
+	public static bool HasHand(GameObject player)
+	{
+		LivingHealthMasterBase bodyParts = PlayerManager.LocalPlayerScript.GetComponent<LivingHealthMasterBase>();
```

Comment:
```suggestion
		if(player.TryGetComponent<LivingHealthMasterBase>(out var bodyParts)
```
You weren't used the passed argument anywhere and if for whatever reason the component was null, it would fail.